free-1.md

```diff
@@ -1,50 +1,61 @@
 
 ### Domain
 _Describe the project's domain in five words._
 
+User-generated music and animations 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+If I were to make an app that plays music in real time, the user could "interact with the langauge" through keystrokes. An alternative on a website could be through point-and-click but this may stifle creativity as a keyboard offers so many different options and would generally be more fun to play with. If it were a mobile app, touch could also be an option.
+
+Another option could be to have the music be generated from text. Users could type words, sentences or entire paragraphs which would be parsed, interpreted then played as notes (length of the word could relate to how long the note is, and the first letter of the word could relate to the pitch of the note).  
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+What happens when the program runs would differ depending on if I want the notes to be played in real-time or interpreted from a set of words. There would be no errors as all inputs from the keyboard would be fair game. 
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to make music and animations sequences in this language. But impossible to solve tasks beyond this such as those involving data structures, algorithms or writing proofs. 
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+My inspiration, http://patatap.com/, has done a very good job already but there are a number of things that I'd do differently if I were to choose to pursue this project. For one, it would be cool to have a way to record sound and animation sequences. Currently patatap just allows you to play a sound with an accompanying animation - a visual piano if you will. In my version, I could have a "playground" mode where users could experiment before playing with the "recording" mode. In the "recording" mode, I could record certain sequences which would repeat in a loop, then allow users to add "layers" of sound and animation sequences on top. These layers can be added and deleted. As a super-stretch goal, I could allow users to send save and send recordings they've made. 
 
```

Comment:
Ooh, I really like the idea of being able to record and loop to create ever more complex layers of rhythm and visuals.  (Be careful though, when you make it possible to record sequences, you may very well let in the ability to create data structures! :) )